examples/step-10/step-10.cc

```diff
@@ -32,6 +32,7 @@
 #include <deal.II/dofs/dof_accessor.h>
 #include <deal.II/fe/fe_nothing.h>
 #include <deal.II/fe/fe_values.h>
+#include <deal.II/base/exceptions.h>
```

Comment:
You likely don't need this include file at all. In the end, `step-51` is the only one that currently includes it.

Reply:
Yes, it looks like you are right. Step-5 has not included this. I referred to the `Assert` macro, and it said that it was defined in line `xx` in `exceptions.h`

Reply:
I'd just remove the `#include`. It's one of those files that is included in essentially every other header file.

Reply:
Ok. Got it.